pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -100,6 +100,10 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, c *ConverterContext)
 				}
 			}
 		}
+
+		if iface.State == v1.InterfaceStateLinkDown && iface.SRIOV == nil {
+			domainIface.LinkState = &api.LinkState{State: "down"}
+		}
```

Comment:
If execution had reached this line, this is surely not an SR-IOV NIC, please see 
https://github.com/kubevirt/kubevirt/blob/77181abe699373c4b4bb5bebc5a13d4915f116b2/pkg/virt-launcher/virtwrap/converter/network.go#L51-L53